Jenkinsfile

```diff
@@ -577,19 +577,23 @@ pipeline {
                             filename 'utils/docker/Dockerfile.el.8'
                             label 'docker_runner'
                             additionalBuildArgs dockerBuildArgs(repo_type: 'stable',
+                                                                deps_build: true,
+                                                                parallel_build: true,
                                                                 qb: true) +
-                                " -t ${sanitized_JOB_NAME}-el8 " +
-                                ' --build-arg BULLSEYE=' + env.BULLSEYE +
-                                ' --build-arg QUICKBUILD_DEPS="' +
-                                quickBuildDeps('el8', true) + '"' +
-                                ' --build-arg REPOS="' + prRepos() + '"'
+                                                " -t ${sanitized_JOB_NAME}-el8 " +
+                                                ' --build-arg BULLSEYE=' + env.BULLSEYE +
+                                                ' --build-arg QUICKBUILD_DEPS="' +
+                                                quickBuildDeps('el8', true) + '"' +
+                                                ' --build-arg REPOS="' + prRepos() + '"'
```

Comment:
I think this indentation should all be aligned. Either:
```
                            additionalBuildArgs dockerBuildArgs(repo_type: 'stable',
                                                                deps_build: true,
                                                                parallel_build: true,
                                                                qb: true) +
                                                                " -t ${sanitized_JOB_NAME}-el8 " +
                                                                ' --build-arg BULLSEYE=' + env.BULLSEYE +
                                                                ' --build-arg QUICKBUILD_DEPS="' +
                                                                quickBuildDeps('el8', true) + '"' +
                                                                ' --build-arg REPOS="' + prRepos() + '"'
```
Or:
```
                            additionalBuildArgs dockerBuildArgs(
                                repo_type: 'stable',
                                deps_build: true,
                                parallel_build: true,
                                qb: true) +
                                " -t ${sanitized_JOB_NAME}-el8 " +
                                ' --build-arg BULLSEYE=' + env.BULLSEYE +
                                ' --build-arg QUICKBUILD_DEPS="' +
                                quickBuildDeps('el8', true) + '"' +
                                ' --build-arg REPOS="' + prRepos() + '"'
```

Reply:
Thanks for the comment.

Reply:
@daltonbohning that's not right either, lines 2-4 are parameters to dockerBuildArgs(), after that should be at the same level so I think it was right before your comment.

Reply:
Ah, you're right! Thanks @ashleypittman and please ignore @dinghwah :)